internal/resource/obcluster/utils.go

```diff
@@ -0,0 +1,14 @@
+package obcluster
+
+import "github.com/oceanbase/ob-operator/api/v1alpha1"
+
+func (m *OBClusterManager) doStorageSizeExpand(obzone *v1alpha1.OBZone) bool {
```

Comment:
doStorageSizeExpand is a little bit confused, it feels like actually do the size expantion